workers/unity/Assets/Playground/Scripts/Metrics/MetricSendSystem.cs

```diff
@@ -0,0 +1,75 @@
+using System.Collections.Generic;
+using Unity.Entities;
+using UnityEngine;
+using Improbable.Gdk.Core;
+
+namespace Playground
+{
+    public class MetricSendSystem : ComponentSystem
+    {
+        private WorkerBase worker;
+
+        private float timeElapsedSinceUpdate = 0.0f;
+
+        private List<float> fpsMeasurements = new List<float>();
+
+        protected override void OnCreateManager(int capacity)
+        {
+            base.OnCreateManager(capacity);
+            worker = WorkerRegistry.GetWorkerForWorld(World);
+        }
+
+        protected override void OnUpdate()
+        {
+            if (worker.Connection == null)
```

Comment:
Can inject a worker entity with an IsConnected tag for this too. Not sure which way is better.

Reply:
This is consistent with the implementations of similar systems.